{"uuid": "b784e358-c864-46ca-b5a9-7c09f79f675b", "children": ["9eb5dc88-cd61-4f05-9220-6d37d34b226f"], "befores": [{"name": "app", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: 'geckodriver' executable needs to be in PATH. \n\n", "trace": "  File \"c:\\users\\needl\\pycharmprojects\\python\\env\\lib\\site-packages\\pluggy\\callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"c:\\users\\needl\\pycharmprojects\\python\\env\\lib\\site-packages\\_pytest\\fixtures.py\", line 1117, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"c:\\users\\needl\\pycharmprojects\\python\\env\\lib\\site-packages\\_pytest\\fixtures.py\", line 924, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"C:\\Users\\needl\\PycharmProjects\\python\\conftest.py\", line 30, in app\n    fixture = App(browser=browser, base_url=web_config[\"baseUrl\"])\n  File \"C:\\Users\\needl\\PycharmProjects\\python\\fixture\\app.py\", line 11, in __init__\n    self.wd = webdriver.Firefox()\n  File \"c:\\users\\needl\\pycharmprojects\\python\\env\\lib\\site-packages\\selenium\\webdriver\\firefox\\webdriver.py\", line 164, in __init__\n    self.service.start()\n  File \"c:\\users\\needl\\pycharmprojects\\python\\env\\lib\\site-packages\\selenium\\webdriver\\common\\service.py\", line 83, in start\n    os.path.basename(self.path), self.start_error_message)\n"}, "start": 1597352774314, "stop": 1597352774318}], "start": 1597352774314, "stop": 1597352774395}